xml/cap_admin_minibroker.xml

```diff
@@ -0,0 +1,291 @@
+    </para>
+<screen>&prompt.user;echo > <replaceable>redis.json</replaceable> '[{ "protocol": "tcp", "destination": "<replaceable>10.0.0.0/8</replaceable>", "ports": "<replaceable>6379</replaceable>", "description": "<replaceable>Allow Redis traffic</replaceable>" }]'
+&prompt.user;cf create-security-group <replaceable>redis_networking</replaceable> <replaceable>redis.json</replaceable>
+&prompt.user;cf bind-security-group <replaceable>redis_networking</replaceable> <replaceable>org</replaceable> <replaceable>space</replaceable>
+</screen>
+    <para>
+     Use following ports to define your ASG for a given service:
+    </para>
+    <informaltable>
+     <tgroup cols="2">
+      <thead>
+       <row>
+        <entry>Service</entry>
+        <entry>Port</entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry>MariaDB</entry>
+        <entry>3306</entry>
+       </row>
+       <row>
+        <entry>MongoDB</entry>
+        <entry>27017</entry>
+       </row>
+       <row>
+        <entry>PostgreSQL</entry>
+        <entry>5432</entry>
+       </row>
+       <row>
+        <entry>Redis</entry>
+        <entry>6379</entry>
+       </row>
+      </tbody>
+     </tgroup>
+    </informaltable>
+   </step>
+   <step>
+    <para>
+     Create an instance of the Redis service. The <command>cf
+     marketplace</command> or <command>cf marketplace -s redis</command>
+     commands can be used to see the available plans for the service:
+    </para>
+<screen>&prompt.user;cf create-service <replaceable>redis</replaceable> <replaceable>4-0-10</replaceable> <replaceable>redis-example-service</replaceable></screen>
+    <para>
+     Monitor the progress of the pods and wait until all pods are in a ready
+     state. The example below shows the additional <literal>redis</literal>
+     pods with a randomly generated name that have been created in the
+     <literal>minibroker</literal> namespace:
+    </para>
+<screen>&prompt.user;watch -c 'kubectl get pods --namespace minibroker'
+NAME                                            READY     STATUS             RESTARTS   AGE
+alternating-frog-redis-master-0                 1/1       Running            2          1h
+alternating-frog-redis-slave-7f7444978d-z86nr   1/1       Running            0          1h
+minibroker-minibroker-5865f66bb8-6dxm7          2/2       Running            0          1h
+</screen>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.cap.minibroker-application-usage">
+  <title>Using Minibroker with Applications</title>
+
+  <para>
+   This section demonstrates how to use Minibroker services with your
+   applications. The example below uses the Redis service instance created in
+   the previous section.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Obtain the demo application and use <command>cf push</command> with the
+     <literal>--no-start</literal> flag to deploy the application without
```

Comment:
I suggest "Obtain the demo application from GitHub"...maybe I'm slow, but it helps me :)

Reply:
Updated in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68